.travis.yml

```diff
@@ -37,6 +37,10 @@ matrix:
   - os: linux
     compiler: gcc
     env: RUN_TESTS_ARGS="--cross ubuntu-armhf.txt --cross linux-mingw-w64-64bit.txt" MESON_ARGS="--unity=on"
+  # Test long commandline handling codepath
+  - os: linux
+    compiler: gcc
+    env: MESON_RSP_THRESHOLD=0
```

Comment:
Please hijack an existing job for this (as you did on Azure), since we only have 5 concurrent jobs on Travis.

Reply:
Done.